style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+.toolbar {
+  display: flex;
+  align-items: center;
+  margin-top: 70px;
+  margin-left: 26px;
+}
+
+.mylogo {
+  align-content: center;
+}
+
+.menu {
+  margin-left: 300px;
+}
+
+.headline {
+  margin-top: 40px;
+}
+
+#container-one {
+  display: flex;
+  flex-direction: row;
+}
+
+.first {
+  display: flex;
+  flex-direction: column;
+}
+
+.rect-6 {
+  width: 15px;
+}
+
+.ell-11 {
+  margin-top: -30px;
+}
+
+.second {
+  display: flex;
+  flex-direction: column;
+  margin-left: 100px;
+  margin-top: 70px;
+}
+
+.ell-5 {
+  width: 11px;
+  height: 10px;
+  margin-top: -10px;
+}
+
+.ell-8 {
+  margin-left: 130px;
+}
+
+.fourth {
+  display: flex;
+  flex-direction: column;
+  margin-left: 85px;
+}
+
+.ell-3 {
+  width: 60px;
+  margin-left: 50px;
+  margin-top: -45px;
+}
+
+.introduction {
+  font-size: 65px;
+}
+
+.profession {
+  font-size: 25px;
+  word-wrap: break-word;
+}
+
+.motto {
+  font-size: 25px;
+  color: #7f8cff;
+}
+
+#container-two {
+  margin-left: 20px;
+  margin-top: -120px;
+}
+
+@media (min-width: 675px) {
+  body {
+    background-color: beige;
```

Comment:
- [ ] Please use white color for your website color to follow Figma's design color
![Screenshot from 2023-03-24 11-02-31](https://user-images.githubusercontent.com/91142432/227504615-cb0d5b3a-1175-4dec-956d-34a8ac650307.png)